src/components/App/App.js

```diff
@@ -11,11 +11,14 @@ import { LoggedInContext } from "../../contexts/loggedInContext";
 import { IsMenuOpenContext } from "../../contexts/isMenuOpenContext";
 import { IsLoginOpenContext } from "../../contexts/isLoginModalOpen";
 import { CurrentUserContext } from "../../contexts/currentUserContext";
+import { SuccessRegisterContext } from "../../contexts/successRegisterContext";
 import SavedNews from "../SavedNews/SavedNews";
 import RegisterSuccessModal from "../RegisterSuccessModal/RegisterSuccessModal";
 import LoginRegisterModal from "../LoginRegisterModal/LoginRegisterModal";
 import Footer from "../Footer/Footer";
 import HeaderGroup from "../HeaderGroup/HeaderGroup";
+import * as auth from '../../utils/authRequests';
+import Api from "../../utils/Api";
 
 function App() {
   let location = window.location.pathname;
```

Comment:
`let location = window.location.pathname;`

you could have used `useLocation` hook